geolocator_platform_interface/lib/src/implementations/method_channel_geolocator.dart

```diff
@@ -90,6 +90,12 @@ class MethodChannelGeolocator extends GeolocatorPlatform {
     }
   }
 
+  @override
+  Future<LocationAccuracyStatus> getLocationAccuracy() async {
+    final accuracy = await _methodChannel.invokeMethod('getLocationAccuracy');
+    return LocationAccuracyStatus.values[accuracy as int];
```

Comment:
I think (but you'd have to try this) that if you specify the type you don't need the cast:

```suggestion
    final int accuracy = await _methodChannel.invokeMethod('getLocationAccuracy');
    return LocationAccuracyStatus.values[accuracy];
```